assembl/static2/css/variables.scss

```diff
@@ -162,26 +162,26 @@ $screen-lg-min: 1200px;
 // Proper media-queries examples https://stackoverflow.com/questions/18424798/twitter-bootstrap-3-how-to-use-media-queries
 
 /* Large desktops and laptops */
-@media (min-width: $screen-lg-min) {
-
-}
+// @media (min-width: $screen-lg-min) {
```

Comment:
uhuh

Reply:
Why is this commented?

Reply:
stylelint doesn't like empty rules. We can remove it completly too.